src/main/java/entity/User.java

```diff
@@ -1,19 +1,25 @@
 package entity;
 
 import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
 
 public class User {
     private String userName;
     private String userEmail;
     private String userPassword;
     private LocalDateTime createdAt;
+    private List<Recipe> recipes;
```

Comment:
what is recipes?

Reply:
recipes associated with the users (recipes that the user has saved)

Reply:
sounds good - note for the future then: a better variable name might be good (like savedRecipes, or something like that)